### DISCUSSION TITLE
SPELL?

### DISCUSSION THREAD

**Discussion by ywlaw** [yamcs/yamcs#641] [General]
Dear SAS team, firstly, thanks for open-sourcing yamcs.

According to your [2018 SpaceOps conference paper](https://arc.aiaa.org/doi/10.2514/6.2018-2682), SES' Satellite Procedure Execution Language and Library (SPELL) has been integrated with yamcs. Would you be able to confirm this? I looked through your yamcs source code but I'm not sure. I also notice SES has a [fork of your yamcs repo](https://github.com/Spacecraft-Code/yamcs).

Thanks in advance for your reply.

**Comment by xpromache** [comment]
Indeed, SPELL has been integrated with Yamcs and is being used in some
operations centers.

However we did not release that publicly because we did not want to get
support requests on the open-source forums. It also requires a bit more
polishing to make it easier to use.

But you can contact us on SpaceApps (yamcs at spaceapplications.com) if you
are interested.

**Reply by ywlaw** [reply]
Thanks for your quick reply, Nicolae. I'll be using Yamcs purely for academic research. Hopefully I'll be able to give back to you guys if I end up making any meaningful contribution. Wish you a great year ahead.
